sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveSerDeSuite.scala

```diff
@@ -82,6 +82,11 @@ class HiveSerDeSuite extends HiveComparisonTest with PlanTest with BeforeAndAfte
     }.head
   }
 
+  // Some tests are failed in Jenkins presumably due to the similar reason described
+  // in SPARK-30798.
```

Comment:
maybe say it more explicitly: `make sure we set the config values to TestHive.conf`